src/imgui/DebuggableEditor.cc

```diff
@@ -338,10 +297,10 @@ void DebuggableEditor::drawExport(const Sizes& s, Debuggable& debuggable)
 			if (ImGui::Button("Export")) {
 				try {
 					auto output = (exportFormatted == EXPORT_FORMATTED)
-						? formatToString(
-							debuggable, *begin, *end, exportPrefix, exportColumns,
+						? formatToString([&](unsigned address) { return debuggable.read(address); },
+							*begin, *end, exportPrefix, exportColumns,
 							exportSuffix, formatStr, interp)
-						: rawToString(debuggable, *begin, *end);
+						: rawToString([&](unsigned address) { return debuggable.read(address); }, *begin, *end);
```

Comment:
Maybe a detail: factor out the (now repeated) fetch function. And pass this to `formatToString()` and `rawToString()`.
```
auto fetch = [&](unsigned addr) { return debuggable.read(addr); };
...
? formatToString(fetch, ...);
: rawToString(fetch, ...);
```